app/src/main/kotlin/com/mapbox/vision/teaser/view/PlaybackSeekBarView.kt

```diff
@@ -0,0 +1,38 @@
+package com.mapbox.vision.teaser.view
+
+import android.content.Context
+import android.text.format.DateUtils
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import android.widget.SeekBar
+import com.mapbox.vision.teaser.R
+import kotlinx.android.synthetic.main.view_playback_seek_bar.view.*
+
+class PlaybackSeekBarView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyle: Int = 0,
+    defStyleRes: Int = 0
+) : LinearLayout(context, attrs, defStyle, defStyleRes) {
+
+    init {
+        LayoutInflater.from(context).inflate(R.layout.view_playback_seek_bar, this, true)
+    }
+
+    var onSeekBarChangeListener: SeekBar.OnSeekBarChangeListener? = null
+        get() = field
+        set(value) {
+            seek_bar.setOnSeekBarChangeListener(value)
+        }
+
+    fun setTimePosition(seconds: Float) {
```

Comment:
Suggestion : call it `setProgress` for compatibility with VisionReplayManager.getProgress?

Reply:
yeah, let's use the same name